### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by ulope**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,15 +50,15 @@ may also provide configurations with services split among multiple servers.
...
++---------+---------+
+          |https://
+==========|==========
+          |
++---------v---------+                       Federation to
 |                 +-+-------------------->  other Raiden
 |      Traefik    | |                       Matrix servers
-|                 | |
-+---------+-------+-+---------+
+|                 |-+---------+
```
Comment (karlb) [inline README.md:60]: The Monitoring Service (or to be precise, its companion process the Request Collector) also reads Matrix messages. Unsurprisingly, it is interested in the MonitoringRequest messages.

File: README.md [hunk 2]
```diff
@@ -147,34 +154,47 @@ Note: The default Postgres configuration assumes 16GiB of system RAM
...
      (to do that uncomment the default values of the `CIDR_ALLOW_METRICS` and `CIDR_ALLOW_PROXY` settings).
    - We also recommend that you provide your own monitoring. The setup of which is currently out of scope of this document.
-   - Please, read carefully the disclaimers for the path finding and monitoring service and uncomment the variables `XX_ACCEPT_DISCLAIMER` if you agree. Note, that the services won't start if you do not.
-1. If you haven't done so before, run `./register-service-provider.sh` (it uses configuration values from `.env`). Please read the information provided [Registering as a RSB Provider](#installation/registering) carefully before executing the script.
+   - Please read the disclaimers for the path finding and monitoring services carefully and uncomment the variables `<SERVICE>_ACCEPT_DISCLAIMER` if you agree. Note, that without agreement the services won't start.
+1. If you haven't done so before, run `./register-service-provider.sh` (it uses configuration values from `.env`). Please read the information provided [Registering as a RSB Provider](#registering-as-a-rsb-provider-) carefully before executing the script.
```
Comment (karlb) [inline README.md:158]: Is the trailing dash at `#registering-as-a-rsb-provider-` correct?

File: README.md [hunk 3]
```diff
@@ -147,34 +154,47 @@ Note: The default Postgres configuration assumes 16GiB of system RAM
...
-  - Make sure that the account is stored in `${DATA_DIR}/keystore/`. If not, the script will exit with an error and you cannot register as a service provider.
-  - Make sure, that the account, configured in `KEYSTORE_FILE`, has enough funding to register as a service provider. Click [here](https://goerli.etherscan.io/address/0x735722704e365c37247bb3e4ec52d6c937c54539#readContract) (Currently deployed on goerli) to find out what the current price for a slot is. You will find the price under `3. currentPrice` denominated in SVT. Otherwise, the script will tell you the price, too.
+  - Make sure that you have configured a keystore file (`$KEYSTORE_FILE` in `.env`). If not, the script will exit with an error and you cannot register as a service provider.
+  - Make sure that the configured account has enough funding to register as a service provider. 
+    You can check the [registry contract](https://goerli.etherscan.io/address/0x735722704e365c37247bb3e4ec52d6c937c54539#readContract) (Currently deployed on goerli) for the current price of a slot. 
+    You will find the price under `3. currentPrice` denominated in SVT.
```
Comment (karlb) [inline README.md:182]: Well, on mainnet it will be in RDN.

File: README.md [hunk 4]
```diff
@@ -147,34 +154,47 @@ Note: The default Postgres configuration assumes 16GiB of system RAM
...
 
 ### Verifying that the RSB is working
 
+Check the status of the services by executing `docker-compose ps`. 
+If any services are in a state other than `Up`, `Up (healthy)` or `Exit 0` after the elapse of the 24h waiting period a configuration problem is the most likely cause.  
+See [troubleshooting the RSB installation](#troubleshooting-the-rsb-installation) below in that case.
+
 - Matrix
 
 - PFS
```
Comment (fredo) [inline README.md:200]: Check the endpoint `https://pfs.${SERVER_NAME}/api/v1/info` for success

File: README.md [hunk 5]
```diff
@@ -202,7 +221,7 @@ docker-compose up -d
 ## Notes:
 
  - A 'purge' script/service will run once a day, pruning old state from rooms to save disk space,
-   and restarting the ``synapse`` service to fetch an up-to-date whitelist of servers.
+   and restarting the ``synapse`` service to fetch an up-to-date whitelist of servers if necessary.
```
Comment (fredo) [inline README.md:225]: "Our" purger only kicks user and fetches the whitelist. The rest is done by the synapse built-in purgers

File: README.md (hunk 1 above)
Comment (ulope) [reply README.md:60]: Ah good point, will update.

File: README.md (hunk 2 above)
Comment (ulope) [reply README.md:158]: Yeah, no idea why github renders it that way :/

File: README.md [hunk 6]
```diff
@@ -147,34 +154,47 @@ Note: The default Postgres configuration assumes 16GiB of system RAM
...
+
+### Registering as an RSB Provider
+For your newly deployed Raiden Service Bundle to be used by Raiden nodes it must be registered.
 
 1. **Registering in the Services Registry On-Chain**  
   - In order to register as a service provider you need to run the script [`register-service-provider.sh`](https://github.com/raiden-network/raiden-service-bundle/blob/master/register-service-provider.sh).
-  - Make sure that the account is stored in `${DATA_DIR}/keystore/`. If not, the script will exit with an error and you cannot register as a service provider.
-  - Make sure, that the account, configured in `KEYSTORE_FILE`, has enough funding to register as a service provider. Click [here](https://goerli.etherscan.io/address/0x735722704e365c37247bb3e4ec52d6c937c54539#readContract) (Currently deployed on goerli) to find out what the current price for a slot is. You will find the price under `3. currentPrice` denominated in SVT. Otherwise, the script will tell you the price, too.
+  - Make sure that you have configured a keystore file (`$KEYSTORE_FILE` in `.env`). If not, the script will exit with an error and you cannot register as a service provider.
```
[the remark this answers is not in the document]
Comment (ulope) [reply README.md:179]: Hm ok, then I think we should probably add a section on correctly setting up the env file and not bury it here near the end of the document.

File: README.md (hunk 3 above)
Comment (ulope) [reply README.md:182]: Lets do that in the next round

Comment (ulope) [reply README.md:182]: See #139

File: README.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (ulope) [reply README.md:179]: See #139

**Pull request merged**